doc_source/lambda.md

```diff
@@ -39,189 +33,166 @@ license = "YOUR-LICENSE"
+                "Successfully stored the incoming request in S3 with the name '{}'",
+                &filename
+            );
+
+            // Return `Response` (it will be serialized to JSON automatically by the runtime)
+            Ok(Response {
+                req_id: event.context.request_id,
+                body: format!(
+                    "the lambda has successfully stored the your request in S3 with name '{}'",
+                    filename
+                ),
+            })
+        }
+        Err(err) => {
             // In case of failure, log a detailed error to CloudWatch.
-            error!(
+            tracing::error!(
                 "failed to upload file '{}' to S3 with error: {}",
-                &filename, err
+                &filename,
+                err
             );
-            // The sender of the request receives this message in response.
-            FailureResponse {
+
+            Err(Box::new(Response {
+                req_id: event.context.request_id,
                 body: "The lambda encountered an error and your message was not saved".to_owned(),
-            }
-        })?;
-
-    info!(
-        "Successfully stored the incoming request in S3 with the name '{}'",
-        &filename
-    );
-
-    Ok(SuccessResponse {
-        body: format!(
-            "the lambda has successfully stored the your request in S3 with name '{}'",
-            filename
-        ),
-    })
+            }))
+        }
+    }
 }
-```
-
-## Step 3: Package and upload the app<a name="lambda-step3"></a>
 
-This topic shows you how to package your app and upload it\.
-
-To be usable as a Lambda, your Rust app must be compiled for the **x86\_64\-unknown\-linux\-musl** target\. This typically means cross compiling the app\. You might encounter build errors if your app needs a certain dependency, but that dependency isn’t available on your system\.
-
-This section shows two approaches to cross compiling that you can try\.
-
-### Rustup approach<a name="lambda-step3-x1"></a>
-
-This approach uses the [Rustup](https://rustup.rs) toolchain to cross\-compile your app\.
-
-**Note**  
-If you’re developing on a Mac, you must install the MUSL build tooling provided by [musl\-cross](https://github.com/FiloSottile/homebrew-musl-cross) and expose it as described in [this issue](https://github.com/awslabs/aws-sdk-rust/issues/186#issuecomment-898442351)\.
-
-1. Install the **x86\_64\-unknown\-linux\-musl** toolchain with **Rustup** by running:
-
-   ```
-   rustup target add x86_64-unknown-linux-musl
-   ```
-
-1. Build the app by running:
-
-   ```
-   cargo build --release --target x86_64-unknown-linux-musl
-   ```
-
-If this approach fails, try the next one\.
-
-### Container approach<a name="lambda-step3-x2"></a>
-
-This approach requires that you install either [Docker](https://www.docker.com) or [Podman](https://podman.io)\.
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    tracing_subscriber::fmt()
```

Comment:
I almost want a default tracing_subscriber with these settings for Lambda... I'll copy this to the docs examples, thanks!

(No action needed)